### PR TITLE
SAK-46828 set default pageSizes of 20,100,200 for use by JSF pagers

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

File: jsf/jsf-widgets/src/java/org/sakaiproject/jsf/bundle/pager_en.properties [hunk 1]
```diff
@@ -1,24 +1,8 @@
...
-pager_textPageSizeAll=all
 
-pager_titleFirst=First {0} {1}
-pager_titlePrev=Previous {0} {1}
-pager_titleNext=Next {0} {1}
-pager_titleLast=Last {0} {1}
-
-# pager widget configuration  values
-pager_pageSize=20
-pager_pageSizes=5,10,20,50,100
```
Comment (ottenhoff) [inline jsf/jsf-widgets/src/java/org/sakaiproject/jsf/bundle/pager_en.properties:21]: 5 is too small for a pager

also moved these into main pager.properties

**Review by ern — APPROVED** [review]

**Review by bjones86 — APPROVED** [review]

**Review by jesusmmp — APPROVED** [review]
Looks good!

I don't understand yet why in Spanish it works correctly without this patch...

**Comment by ottenhoff** [comment]
Thanks. My guess is it works because of these defaults:

`
 95         // get stuff for page size selection and display
 96
 97         String textPageSize = getString(context, component, "textPageSize", "Show {0}");
 98         String textPageSizeAll = getString(context, component, "textPageSizeAll", "all");
 99         String pageSizesStr = getString(context, component, "pageSizes", "5,10,20,50,100");
`

**Comment by ottenhoff** [comment]
maybe i should change this as well.....

**Pull request merged**